Feedback/templates/Feedback/teacher_check.html

```diff
@@ -4,45 +4,63 @@
 {%load static%}
 
 {% block feedback %}
-    <link rel="stylesheet" type="text/css" href="{% static 'Feedback/teacher_check.css' %}">
+  <link rel="stylesheet" type="text/css" href="{% static 'Feedback/teacher_check.css' %}">
+  <h1 align="center"class="check-h1">Feedback Details</h1><br><br>
+  <form action="{%url 'teacher_check'%}" method="POST" class="check-form">
+    {%csrf_token%}
+	  <div class="form">		
+	    <h3>
+		  Department:-
+			<input list="teacher_dept" name ="dropdown" autocomplete="off">
+			  <datalist id="teacher_dept"class="check-datalist">
+				<option selected="disabled" value="none"></option>
+				  {% for element in teacher %}
+					<option value="{{ element.teacher_dept }}"></option>
+				  {%endfor%}
+			  </datalist>
+			</input>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
+		  Faculty:-
+			<input list="teacher_name" name='dropdown1' autocomplete="off">
+			  <datalist id="teacher_name"class="check-datalist">
+				<option selected="disabled" value="none"></option>
+				  {% for element in teacher %}
+					<option value="{{ element.teacher_name }}"></option>
+				  {%endfor%}
+			  </datalist>
+			</input>
+		</h3>
+		</div><br><br>	
+		<button class="btn btn-primary check-button"name="submit" type="submit" name = 'Submit'  >Submit</button><br><br>

```

Comment:
here give space between `class` and `name` property.
Why have you given two `name` values?
And remove space before `>` in opening button tag.